### ISSUE TITLE
how to transformer raw_rna_data/combine/brca to pt file?

### ISSUE THREAD

**Issue by liujun0621** [mahmoodlab/TANGLE#8]
Thank you for your great work.
how to transformer raw_rna_data/combine/brca to pt file?
combine as follows   https://github.com/mahmoodlab/SurvPath/tree/main/datasets_csv/raw_rna_data/combine/brca
# raw data  for example TCGA-WT-AB41  from above csv file
raw = torch.tensor([
0.9115,-1.1811,-1.8836,4.2518,7.7066,1.4756,8.2921,1.0847,1.7097...
], dtype=torch.float32)

# expeced data
expected = torch.tensor([
   0.4037, 0.3137, 0.4114, 0.4151, 0.7911, 0.1754, 0.7278, 0.2572, 0.5961,
        ...
], dtype=torch.float32)

when i use Min-Max Normalization method, the result don't equal expected as follows,could you help me?
